bin/extractTypes.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs')
```

Comment:
A little script to regex-parse out `type`/`interface` definitions from Typescript's builtin `lib.*.d.ts` files since some of them are super long

So this was used to generate `src/typenames/*.coffee`